src/gui/color_picker_proxy.h

```diff
@@ -55,6 +55,8 @@ void dt_iop_color_picker_cleanup();
 /* link color picker to widget */
 GtkWidget *dt_color_picker_new(dt_iop_module_t *module, dt_iop_color_picker_kind_t kind, GtkWidget *w);
 
+guint dt_get_modifier_state();
```

Comment:
I suppose this should be better in `gui/accelerators.h`, no?

And maybe add a bit of comment for the spec.

Reply:
Not really; nothing to do with shortcut keys...

Maybe gtk.c? Just because it already is a collection of functions with complicated gtk dependencies.

Just get_modifier_state? key_modifier_state? dt_key_modifier_state?

Reply:
Ok for gtk.h/c and let's go with `dt_key_modifier_state`. (a public routine should always start with dt_).